lib/Widgets/TransmissionListEntry.vala

```diff
@@ -45,36 +48,44 @@ public class Drop.Widgets.TransmissionListEntry : Gtk.Grid {
     }
 
     private void build_ui () {
-        this.column_spacing = 6;
-        this.row_spacing = 6;
+        this.orientation = Gtk.Orientation.HORIZONTAL;
+        this.spacing = 12;
+
+        main_grid = new Gtk.Grid ();
+        main_grid.column_spacing = 12;
+        main_grid.row_spacing = 2;
 
         icon = new Gtk.Image ();
 
         primary_label = new Gtk.Label ("");
         primary_label.halign = Gtk.Align.START;
+        primary_label.valign = Gtk.Align.END;
         primary_label.ellipsize = Pango.EllipsizeMode.MIDDLE;
-        primary_label.get_style_context ().add_class (Granite.StyleClass.H3_TEXT);
+        primary_label.use_markup = true;
```

Comment:
yeah, it's needed to stay consistent with the other indicators.. Battery, session and music all use <b>